vinniefalco/Beast
# Version 83

* Fix extraneous argument in async_write_header
* Add flat_static_buffer::mutable_data
* Add buffer_front
* Add wstest compression option
* Turn some flat_static_buffer_tests on

WebSocket

* Add wstest compression option
* Fix buffer lifetime in websocket write

API Changes:

* Add static_buffer 

Actions Required:

* Callers who depend on static_buffer returning sequences of
  exactly length one should switch to flat_static_buffer.

Documentation preview:
http://vinniefalco.github.io/stage/beast/v83
